src/services/transaction/index.ts

```diff
@@ -0,0 +1,13 @@
+import { ENDPOINTS } from "@constants/services";
+import { axiosInstance } from "@utils/axiosInstance";
+
+const getTransactions = async ({ pageNumber }: { pageNumber: number }): Promise<[]> => {
```

Comment:
should return `Promise<TTransactionAll[]>

Reply:
This will change the implementation of and also break the test. but the type of the response is explicitly infered from transactionCard component